FSQLocationBroker/FSQLocationBroker.m

```diff
@@ -296,12 +294,10 @@ - (void)refreshLocationSubscribers {
         self.isUpdatingLocation = NO;
     }
     
-#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 90000
     // Should allow background location
-    if ([self.locationManager respondsToSelector:@selector(allowsBackgroundLocationUpdates)]) {
+    if (@available(iOS 9.0, *)) {
```

Comment:
You've changed the behavior here - should probably both check the `@available` and the `respondsToSelector:`